### ISSUE TITLE
Broken configuration files due to Augeas change

### ISSUE THREAD

**Issue by pkruithof** [saltstack/salt#38078]
In PR #34643, a change was made to support quotes in arguments. However this change results in every augeas value being inserted with quotes.

The previous state works in 2016.3.4, but not in 2016.11.0:

```yaml
sshd_config:
  augeas.change:
    - lens: sshd.lns
    - context: /files/etc/ssh/sshd_config
    - changes:
      - set PasswordAuthentication 'no'
```

The problem is that the configuration file ends up with `PasswordAuthentication 'no'` instead of `PasswordAuthentication no`, which breaks SSH logins in this case due to the configuration file being invalid.

**Comment by pkruithof** [comment]
What's interesting is that the tests in the PR fail. I cannot see what tests are failing, because it is not available anymore. However I'm wondering why failing code gets merged in the first place?

**Comment by gtmanfred** [comment]
We have a lot of tests, and they fail from time to time, so sometimes we have to merge with failing tests.  But we do not merge if tests are failing around the change that was made.

**Comment by Ch3LL** [comment]
@pkruithof just to make sure if you add quotes around the arguments does it then work? just making sure you have a workaround set in place until we can get it fixed.

**Comment by pkruithof** [comment]
Not sure, I downgraded Salt to 2016.3.4 to get it working again.

**Comment by Ch3LL** [comment]
K thanks. Looks like the fix was reverted in that PR and will be reviewed/merged. Will be included in 2016.11.1.

**Comment by hoonetorg** [comment]
@Ch3LL https://github.com/saltstack/salt/issues/38115 is not included in 2016.11.4.
It breaks all augeas states in our environment
